blazor/diagram/swimlane/lane/lane.md

```diff
@@ -115,6 +115,83 @@ The following code example explains how to define a lane header and its customiz
 
 You can download a complete working sample from [GitHub]()
 
+Lane header annotaiton also supports template. you can define a HTML content at tag level and specify the use of template with [UseTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.Annotation.html#Syncfusion_Blazor_Diagram_Annotation_UseTemplate) property. If you want to define separate template for each lane, differentiate the annotaiton by the ID property.
```

Comment:
Lane header annotation also supports the template. You can define HTML content at the tag level and specify the use of a template with the [UseTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.Annotation.html#Syncfusion_Blazor_Diagram_Annotation_UseTemplate) property. To define a separate template for each lane, differentiate the annotation by the ID property.